packages/core-utils/src/itinerary.js

```diff
@@ -26,6 +26,23 @@ export function isTransit(mode) {
   return transitModes.includes(mode) || mode === "TRANSIT";
 }
 
+export function isOptional(leg) {
```

Comment:
Rename to `isReservationRequired` or `isBookingRequired` (to match the return condition).

Reply:
Renamed in f7aec8908db009cc5d1b7ddbfe2e062539692f51